src/open_inwoner/accounts/admin.py

```diff
@@ -60,6 +60,10 @@ class _UserAdmin(UserAdmin):
                 ),
             },
         ),
+        (
+            _("Contacts - invites"),
+            {"fields": ("user_contacts", "contacts_for_approval")},
```

Comment:
I would also add these to filter_horizontal (or perhaps raw_id_fields). The default M2M admin widget in Django is annoying and shouldn't be used imho.